### ISSUE TITLE
unoconv: Cannot find a suitable pyuno library and python binary combination in /usr/lib64/libreoffice

### ISSUE THREAD

**Issue by andreabravetti** [unoconv/unoconv#132]
With Fedora 19 default libreoffice 4.0.3 and pyuno installation and obviously with the patch "python3: added compatibility" from tanty:

unoconv: Cannot find a suitable pyuno library and python binary combination in /usr/lib64/libreoffice
ERROR: No module named uno

It seems that they decided to install uno.py and unohelper.py under /usr/lib64/python3.3/site-packages instead of /usr/lib64/libreoffice/program...

I do not know if it is a bug in fedora or unoconv

**Comment by regebro** [comment]
Duplicates #49

**Comment by ranyuye** [comment]
Please, i want to know this trouble details, because i got the same trouble on debian os, thanks.

**Comment by regebro** [comment]
To find which Python to use to run unoconv, you can try a script I made.
```

cd /tmp
wget -l https://gist.githubusercontent.com/regebro/036da022dc7d5241a0ee97efdf1458eb/raw/1bc0655423d196acd79a5d9fa60d2baada8dd534/find_uno.py
python3 find_uno.py

```
It should list all Pythons that have a Libreoffice installed.

**Comment by ranyuye** [comment]
Thanks, I found the python that I was actually running unconv after executing the py file. I was wondering if this error has anything to do with my using Docker to deploy my project? Can you give me some advice?

**Comment by regebro** [comment]
Unoconv should be run with the same Python as Libreoffice is using. If you use system distributions of LIbreOffice, that's almost always the system python, and that makes it easier.

**Comment by ranyuye** [comment]
Yes, it worked! I can use it in the console after switching my python path, but I still can't use the python program call, this may be a problem in my program, anyway, thank you for your reply and suggestions, best wishes!

**Comment by ranyuye** [comment]
Good news! After finding the python path and specifying the python path in docker like this, it runs successfully.
`ENV PYTHONPATH=/usr/lib/python3/dist-packages/`, hopefully this will help others with similar issues.
